### PR TITLE
Create feedback default container

### LINKED ISSUE #646: [REFACTOR] Create a default feedback prefab container
Instead of having to add each feedback to each character, we could have a default to apply to all of them.

### PR DESCRIPTION
**Pull request by BertovDev**
This PR adds a new prefab called : `FeedbackContainer` , it includes all the feedbacks prefabs such as poison, slowed, etc.

Also it refactors the way we call this feedbacks, removing the use of heavy function call such as `GameObject.find`.

To add new feedback its pretty simple:

1. Add the new feedback prefab to the `FeedbackContainer` prefab
2. In the editor add the new Prefab to the feedback prefab list in the feedback container script.
3. In `PlayerFeedback` implement the new method to the respective feedback

Finally the PR also implements a new way of activating PlayerStateFeedbacks such as Posioned and Slowed.
Now its pretty simple to add a new feedback , in `Battle.cs` add to the new Enum `StateEffects` the effect name with the same number as it have in PlayerEffects. I will create a Documentation for this.
closes #646

### CODE REVIEW

File: client/Assets/Scripts/Battle.cs [hunk 1]
```diff
@@ -782,8 +789,7 @@ private GameObject findGhostPlayer(string playerId)
         float characterSpeed
     )
     {
-        ManagePoisonedFeedback(player, playerUpdate);
-        ManageSlowedFeedback(player, playerUpdate);
+        ManageFeedbacks(player, playerUpdate);
```
Comment (ncontinanza) [inline client/Assets/Scripts/Battle.cs:792]: Neat ⭐

**Review by ncontinanza — DISMISSED** [review]
Nice job!

**Review by AngieDutra — DISMISSED** [review]
Really like this refactor! 🎆

**Review by mgrunwaldt — CHANGES_REQUESTED** [review]
Great job, just a tiny suggestion

**Review by mgrunwaldt — DISMISSED** [review]

**Review by mgrunwaldt — APPROVED** [review]

**Review by AngieDutra — APPROVED** [review]
LGTM ✨

**Pull request merged**